src/tests/ftest/slurm_setup.py

```diff
@@ -66,9 +90,13 @@ def update_config_cmdlist(args):
 
     """
     all_nodes = NodeSet("{},{}".format(str(args.control), str(args.nodes)))
-    cmd_list = ["sed -i -e 's/ClusterName=cluster/ClusterName=ci_cluster/g' {}".format(SLURM_CONF),
-                "sed -i -e 's/SlurmUser=slurm/SlurmUser={}/g' {}".format(args.user, SLURM_CONF),
-                "sed -i -e 's/NodeName/#NodeName/g' {}".format(SLURM_CONF)]
+    if create_epilog_script(args) > 1:
+        logger.error(f"% {EPILOG_FILE} could not be updated. Check if file exists")
```

Comment:
Pylint is complaining
`logging-fstring-interpolation, Use lazy % formatting in logging functions`
```suggestion
        logger.error("could not be updated. Check if file exists", EPILOG_FILE)
```